css/properties/flex-grow.json

```diff
@@ -0,0 +1,120 @@
+            "edge_mobile": [
+              {
+                "version_added": true
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "firefox": [
+              {
+                "version_added": "20",
+                "notes": "Since Firefox 28, multi-line flexbox is supported."
+              },
+              {
+                "version_added": "18",
+                "version_removed": "20",
+                "flag": {
+                  "type": "preference",
+                  "name": "layout.css.flexbox.enabled",
+                  "value_to_set": "true"
+                }
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "20",
+                "notes": "Since Firefox 28, multi-line flexbox is supported."
+              },
+              {
+                "version_added": "18",
+                "version_removed": "20",
+                "flag": {
+                  "type": "preference",
+                  "name": "layout.css.flexbox.enabled",
+                  "value_to_set": "true"
+                }
+              }
+            ],
+            "ie": {
+              "alternative_name": "-ms-flex-positive",
+              "version_added": "11"
+            },
+            "ie_mobile": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "12.10"
+            },
+            "opera_android": {
+              "version_added": "12.10"
+            },
+            "safari": {
+              "prefix": "-webkit-",
+              "version_added": "6.1"
+            },
+            "safari_ios": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "less_than_zero_animate": {
+          "__compat": {
+            "description": "<0 animate",
+            "support": {
+              "chrome": {
+                "version_added": "49"
+              },
+              "firefox": {
+                "version_added": "32",
+                "notes": "Before Firefox 32, Firefox wasn't able to animate values starting or stopping at <code>0</code>."
+              },
+              "firefox_android": {
+                "version_added": "32",
+                "notes": "Before Firefox 32, Firefox wasn't able to animate values starting or stopping at <code>0</code>."
+              },
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              }
+            }
```

Comment:
Status is missing. Is this a standard behaviour?

Reply:
The spec says "Negative numbers are invalid", so I've added a status statement. Just guessing on being deprecated though.